### ISSUE TITLE
marginnote in \footnoteA

### ISSUE THREAD

**Issue by heinrichffleck** [maieul/ledmac#684]
Bonsoir Maieul,

est-il possible insérer notes au margin in ''environment'' \footnoteA?

Si je  écris (reledmac et reledpar)

\pstart
text text text\fotnoteA{\ledouternote{text} text text test}
text text text
\pend
pendant la compilation j'ai

! Package reledmac Error: \edtext outside numbered paragraph (\pstart\IeC {\tex
tellipsis }\pend). 

e j'ai imprimé dans la note in bold:
 **(Edtext outside numbered paragraph)**

Merci pour la reponse, saluti, Enrico

**Comment by maieul** [comment]
non, ce n’est pas possible. Cela fait parti des choses que je dois faire depuis longtemps…

mais comme tout le monde m’ajoute des nouvelles demandes j’ai jamais le tps de m’en occuper

**Comment by Doc73** [comment]
> tout le monde

@maieul se réfère à moi! Ahahahah! :-D

**Comment by maieul** [comment]
@doc73 est effectivement des utilisateurs les plus requetant, mais pas le seul

**Comment by heinrichffleck** [comment]
Merci egalment, bon soir

**Comment by heinrichffleck** [comment]
J'ai suppli aussi:

\footnoteA{\hspace*{-20mm}text in margin}
Saluti, Enrico

**Comment by heinrichffleck** [comment]
ERRATA CORRIGE
Pardon, j'ai obliee d'ecrire \usepackage{marginnote}:

\marginnote{\hspace*{-20mm}Fgl. 81R+88V}

**Comment by maieul** [comment]
Je pense que cela ne serait pas très compliqué à intégrer proprement… mais il faut du tps

**Comment by heinrichffleck** [comment]
Cher Maieul, 
je pense que si tu aura un jour un beaocoup bocoup de temp, la pemiere chose a faire, selon moi, est penser a reledpar; je me refere a la possibilité d'avoir sur la page de gauche deux text in colonne (en deux languages) e sur la page de droite la tradution.
j'ai,  depuis longtemps, achever la tradution de corps flottants d'Archimede (II livre) et suis, à présente, a la revision, mais n'est pas aussi une bonne edition avec un text en grecque en \footnote et \footnoteA. Un'autre chose est la possibilité de gestion des images flottant dans reledpar: c'est impossible d'utiliser \pstart \pend pour chaque paragraphe parce que à la page successif le nouveau \pstart tombait à la moitiè de la page, ergo, \lednopb, ledpb,...
Ces ne sont pas de critiques. reledpar est execllent.
J'envoi une page de ma version,
Saluti, Enrico
![screenshot](https://user-images.githubusercontent.com/13473498/29429977-eefec20a-8392-11e7-8056-097d6adeac18.jpg)

**Comment by maieul** [comment]
il me faudrait au moins 6 mois à tps plein pour pouvoir faire plus d’un texte en reledpar/reledmac

**Comment by heinrichffleck** [comment]
Je le sais, mieux:  je l'immagine, mais je pense, excuse moi, que plutot que <<arranger>> petites bug ou repondre à petites demendes des usagers, cette  (peut etre) vraiment un objectiv primair.

Moi je ne toi rmercierà(?) suffisament por ton excellent travail de bonne volonté.

Excuse  moi surtout pour maltraiter le français, mais, come une fois, je prfere mal parler en franòais que bien en anglais. Toujours en amitié, Enrico

**Comment by maieul** [comment]
oui, mais les « petits bugs » demandent moins de tps en continue

**Comment by heinrichffleck** [comment]
Oui, ça je le sais!
